.github/workflows/ElastixGitHubActions.yml

```diff
@@ -154,5 +154,20 @@ jobs:
       with: 
         name: "${{ matrix.os }}"
         path: uploads
+        
+    - name: Login to Docker Hub
+      if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags') && matrix.os == 'ubuntu-18.04'
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKER_HUB_USERNAME }}
+        password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+
+    - name: Build and push
+      if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags') && matrix.os == 'ubuntu-18.04'     
+      run: |
+        mkdir output
+        docker build -t superelastix/elastix:${GITHUB_REF:10} .
```

Comment:
This tags with the commit hash, but doesn't automatically build with a new release. Or will release builds be build/pushed manually? I think those are also (perhaps more) interesting for most users.

Reply:
This ensures that a docker image will only be build and pushed to dockerhub on a tagged commit/release. This way dockerhub only contains working and relevant elastix docker images.
I should push release build automatically right?

Reply:
You're right, I missed the `startsWith` condition. Personally I prefer to use `${GITHUB_REF#refs/tags/}` instead of `${GITHUB_REF:10}`, because I think it's a bit clearer as to what it's doing, but that's personal preference.

Reply:
You can also create a seperate 'release' CI, with: 
```
on:
  push:
    tags:
    - '*.*.*'
```
to not clutter your 'nightly' CI.